roles/ceph-rgw-loadbalancer/templates/haproxy.cfg.j2

```diff
@@ -29,7 +29,11 @@ defaults
     maxconn                 8000
 
 frontend rgw-frontend
+{% if haproxy_frontend_ssl_certificate %}
+    bind *:{{ haproxy_frontend_ssl_port }} ssl crt {{ haproxy_frontend_ssl_certificate }}
```

Comment:
I know that's not the initial purpose of this PR but it might be a good idea to add strong ciphers and options (disable ssl v3, etc..) to the TLS configuration via new variables.
WDYT ?

Reply:
Yes that sounds great, we were planning on submitting a separate PR for that but if that's fine with you we can add it here.